lib/correios/cep/address.ex

```diff
@@ -3,7 +3,7 @@ defmodule Correios.CEP.Address do
   Address structure.
   """
 
-  @enforce_keys [:street, :complement, :neighborhood, :city, :state, :zipcode]
+  @enforce_keys [:street, :complement, :neighborhood, :city, :state, :postal_code]
```

Comment:
As a rule to avoid breaking changes, it would be better to add the new field `postal_code` and keep `zipcode` to be removed in a later release. 

On the other hand, if the communication of the breaking changing is effective (README, CHANGELOG, etc), it should not be a pain point for the consumers of the library just rename a field in the expected response. 

@lccezinha Are you consuming Correios CEP? What is your opinion from the library's consumer point of view?

Reply:
Hey, @prodis thanks a lot for your response.

No, I'm not using the library, it just appears to me in the ElixirWeekly 203 and I decide to take a look for issues to contribute.

About the rule to avoid breaking changes, I would keep both fields `zipcode` and `postal_code` for a while, update patch version of the lib and try to add a message to warn about the deprecation of the field `zipcode`.

When you update the minor version I would remove the legacy field `zipcode` to keep only `postal_code`.

Reply:
Please do that.

Reply:
I'll do that :+1:

Reply:
Hey @prodis sorry about the lack of updates, let me ask you a question. Do you suggest add again the field `zipcode` to the code base and also keep the documentation using it, or the documentation must be already using the new `postal_code` field and the old `zipcode` only will be in the code to make the old version compatible?

Reply:
Just add the `zipcode` field back to `Correios.CEP.Address` struct and leave the documentation as it is with `postal_code`. I will update the documentation with the deprecation after merging your pull request.

Reply:
@prodis my last commit broke some stuff could help me?

Reply:
Thanks Dialyzer. 🙌 

https://github.com/prodis/correios-cep-elixir/pull/21#discussion_r436268695